app/scripts/views/settings/clients.js

```diff
@@ -53,6 +53,15 @@ define(function (require, exports, module) {
     _formatAccessTimeAndScope (items) {
       return _.map(items, (item) => {
         item.title = item.name;
+
+        if (item.location) {
+          if (item.location.country && item.location.state) {
+            item.title += ' - ' + item.location.state + ', ' + item.location.country;
```

Comment:
Ah true! yea let's keep in consistent for now, but good contribution for later 👍